### DISCUSSION TITLE
2011-4 - 泛函分析基础：学习指导和习题详解

### DISCUSSION THREAD

**Discussion by giscus** [duskmoon314/THU_BFA#3] [Announcements]
# 2011-4 - 泛函分析基础：学习指导和习题详解

https://duskmoon314.github.io/THU_BFA/ch6/2011-4.html

**Comment by thulanxc** [comment]
第一个小问好像写的和书上定理表述不太一样？

**Reply by duskmoon314** [reply]
$\forall x \in X, f_n(x)$ 为收敛列，收敛至 $f(x)$，那么 $\sup_{n \ge 1} |f_n(x)| < \infty$。

确实敲漏了 orz

**Reply by thulanxc** [reply]
最后一小问极限怎么求出来的呢？为啥是x0/2

**Reply by duskmoon314** [reply]
emmm，好像应该没有 $\frac{1}{2}$，似乎 $n \int_0^{\frac{1}{n}} x(t) dt \approx n \cdot x(0) \cdot \frac{1}{n}$

当时想的是可能类似于 $\delta$ 函数和 $x(t)$ 卷积，但 $\delta$ 是一个 $[-1/n, 1/n]$ 上高 n 的三角的极限，而我只考虑 $[0, 1)$，所以随手写了一个 $1/2$
